manifests/bootkube/bootstrap-manifests/kube-scheduler-pod.yaml

```diff
@@ -0,0 +1,28 @@
+# NOTE: This belongs to scheduler operator, but we need this pass the initial bootstrap now.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: bootstrap-kube-scheduler
+  namespace: kube-system
+  labels:
+    openshift.io/control-plane: "true"
+    openshift.io/component: "scheduler"
+  annotations:
+    openshift.io/run-level: "0"
+spec:
+  containers:
+  - name: kube-scheduler
+    image: {{ .Image }}
+    imagePullPolicy: {{ .ImagePullPolicy }}
+    command: ["/bin/bash", "-c"]
+    args:
+    - exec hyperkube scheduler --config=/etc/kubernetes/config/scheduler.yaml
+    volumeMounts:
+    - mountPath: /etc/kubernetes/secrets
+      name: secrets
+      readOnly: true
+  hostNetwork: true
```

Comment:
does this work? It uses the in-cluster config. I would expect that we also pass the bootstrap kubeconfig.

Reply:
we will have to hardcode the path to it

Reply:
added `--kubeconfig=/etc/kubernetes/secrets/kubeconfig`